import-export-cli/cmd/k8s/addApi.go

```diff
@@ -0,0 +1,230 @@
+package k8s
```

Comment:
Please add licence headers

Reply:
Fixed in fe6321d25fd70225653f8632e3f2b71619642eed